tests/test_config.py

```diff
@@ -22,6 +23,11 @@ def alt_cfg_module():
     return 'config.test_cfg_alt'
 def cfg_file():
     return os.path.join(dirname, "config", "test_cfg.py")
+def pyc_cfg_file():
+    file_path = os.path.join(dirname, "config", "test_cfg.py")
+    pyc_file_path = file_path + 'c'
+    py_compile.compile(file_path, pyc_file_path)
```

Comment:
I think we also need to delete `pyc_file_path` after the test ran. We could use https://docs.pytest.org/en/latest/xunit_setup.html to delete it.